src/test/kotlin/org/jetBrains/research/tasktracker/tracking/Tasks.kt

```diff
@@ -0,0 +1,52 @@
+@file:Suppress("MatchingDeclarationName")
+
+package org.jetBrains.research.tasktracker.tracking
+
+import org.jetBrains.research.tasktracker.models.Extension
+import org.jetBrains.research.tasktracker.tracking.task.Task
+
+data class MockTask(
+    val filename: String,
+    val ext: Extension,
+    val text: String? = null,
+    val relativePath: String? = null
+) : Task {
+    override fun getContent(): String? {
```

Comment:
It is better to use a short notation everywhere here - `override fun getContent() = text`

Reply:
I also would prefer to change the initial Task interface: replace getters with the fields, like: `task. getContent()` -> `task. content`, what do you think?